editor/editor_vcs_interface.cpp

```diff
@@ -418,8 +418,13 @@ void EditorVCSInterface::create_vcs_metadata_files(VCSMetadata p_vcs_metadata_ty
 		if (f.is_null()) {
 			ERR_FAIL_MSG("Couldn't create .gitattributes in project path.");
 		} else {
-			f->store_line("# Normalize EOL for all files that Git considers text files.");
+			f->store_line("# Properly detect languages on Github")
+			f->store_line("*.gd linguist-language=GDScript")
+			f->store_line("")
+			f->store_line("# Normalize EOL for all files that Git considers text files");
 			f->store_line("* text=auto eol=lf");
+			f->store_line("# Except for bat files, which are Windows only files")
```

Comment:
```suggestion
			f->store_line("# Except for Batch files, which are Windows-only files")
```

Reply:
I just took the word of whoever put that in the original here: https://github.com/godotengine/godot/blob/master/.gitattributes

Could remove it if you want.